api/routers/legacy_debug.py

```diff
@@ -86,14 +92,16 @@ async def parse_detection(data:dict) -> dict:
         'equip_hands_id': equipment.get('HANDS'),
         'equip_weapon_id': equipment.get('WEAPON'),
         'equip_shield_id': equipment.get('SHIELD'),
-        'equip_ge_value': data.get('equipment_ge')
+        'equip_ge_value': data.get('equipment_ge', 0)
```

Comment:
Good catch, we could also set equipt g.e. data in the database to zero as default - though this works perfectly well.